tests/integration/create.bats

```diff
@@ -39,3 +39,19 @@ function teardown() {
 
   testcontainer test_busybox running
 }
+
+@test "runc create --pid-file" {
+  runc create --pid-file pid.txt --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  testcontainer test_busybox created
+
+  # check pid.txt was generated
+  [ -e pid.txt ]
```

Comment:
It is not necessary to check that  `pid` matches the pid shown in `runc list`.  but we can check the content of the file as following:

```
run cat pid.txt
[ "status" -eq 0 ]
[[ ${lines[0]} =~ [0-9]+ ]]
```